### PR TITLE
Hlint sources

### PR DESCRIPTION
**Pull request by jsynacek**

### CODE REVIEW

**Comment by arybczak** [comment]
Looks good, but please add accompanying CI check.

**Comment by jsynacek** [comment]
> Looks good, but please add accompanying CI check.

Done.

File: .github/workflows/hlint.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  hlint:
+    runs-on:
+      - ubuntu-22.04
+    steps:
+      # v3.5.3
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
+      # v2.4.6
+      - uses: haskell/actions/hlint-setup@a99601b177e00b98c78b6f6de680a101cf1c619d
+        with:
+          version: 3.5
```
[the remark this answers is not in the document]
Comment (arybczak) [reply .github/workflows/hlint.yaml:29]: Yeah, let's do the latest.

**Review by arybczak — APPROVED** [review]
Looks good, feel free to merge after rebasing and checking that it passes on the current master :)

**Pull request merged**